extensions/default/src/getHangingProtocolModule.js

```diff
@@ -45,6 +45,13 @@ const defaultProtocol = {
             greaterThan: { value: 0 },
           },
         },
+        {
+          attribute: 'isDisplaySetFromUrl',
+          weight: 10,
+          constraint: {
+            equals: true,
+          },
+        },
```

Comment:
I'm not sure why we are weighting things here, we should just always check for all `isDisplaySetFromURL`, and just use that for hanging. We shouldn't add this here. What is the purpose? There is only always one Series and one instance that can get hung first, so we can do it always

Reply:
The initialSeries/SOP parameters are used by the default protocol to choose the initial display set to hang and this is how that gets applied.

Reply:
It has no affect if the user does not supply the parameters.

Reply:
The point of this change is that one can now add custom attributes to do this sort of thing, so it is no longer baked into the core framework, but can be done in various ways as required.  So, the main mode/hanging protocol code no longer needs to know about hanging protocol details around how instances are matched.